README.md

```diff
@@ -1,192 +1,38 @@
-# PyOS Update and Clean Contributor & Peer Review Metadata
+# pyosmeta (was: update-web-metadata)
 
-This repo contains a small module and 3 CLI scripts.
+[![PyPI](https://img.shields.io/pypi/v/pyosmeta.svg)](https://pypi.org/project/pyosmeta/)
```

Comment:
@willingc should we rename the repo to be pyosmeta? 

I actually forgot that i pushed this to pypi. I can also setup VCS versioning and github action based release / publish workflow . i've wanted to create that tutorial anyway so minus well do it using hatchling!